manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -0,0 +1,116 @@
+	// Application name used for deployed objects
+	AppName string `json:"appName"` 
+
+	// admin user initial password
+	ApplicationAdminPassword string `json:"applicationAdminPassword"` 
+
+	// Domain name for the external route
+	// Used for external authentication configuration
+	ApplicationDomain string `json:"applicationDomain"`
+
+	DatabaseName string `json:"databaseName"`
+	DatabasePort string `json:"databasePort"`
+	DatabaseUser string `json:"databaseUser"`
+	DatabasePassword string `json:"databasePassword"`
+	// Application region number
+	DatabaseRegion string `json:"databaseRegion"`
+	// Containerized database volume size
+	DatabaseVolumeCapacity string `json:"databaseVolumeCapacity"`
+
+	HttpdCPUReq string `json:"httpdCPUReq"`
+	HttpdImageName string `json:"httpdImageName"`
+	HttpdImageTag string `json:"httpdImageTag"`
+	HttpdMemLimit string `json:"httpdMemLimit"`
+	HttpdMemReq string `json:"httpdMemReq"`
+
+
+	// memcachedpod deployment information
+	MemcachedCPUReq string `json:"memcachedCPUReq"`
+	MemcachedImageName string `json:"memcachedImageName"`
+	MemcachedImageTag string `json:"memcachedImageTag"` 
+	MemcachedMaxConnection string `json:"memcachedMaxConnection"`
+	MemcachedMaxMemory string `json:"memcachedMaxMemory"`
+	MemcachedMemLimit string `json:"memcachedMemLimit"`
+	MemcachedMemReq string `json:"memcachedMemReq"`
+	MemcachedSlabPageSize string `json:"memcachedSlabPageSize"`
+
+	//  pod deployment information
+	OrchestratorCPUReq string `json:"orchestratorCPUReq"`
+	OrchestratorImageName string `json:"orchestratorImageName"`
+	OrchestratorImageNamespace string `json:"orchestratorImageNamespace"`
+	OrchestratorImageTag string `json:"orchestratorImageTag"`
+	OrchestratorMemLimit string `json:"orchestratorMemLimit"`
+	OrchestratorMemReq string `json:"orchestratorMemReq"`
+
+	// database pod deployment information
+	PostgresqlCPUReq string `json:"postgresqlCPUReq"`
+	PostgresqlImgName string `json:"postgresqlImgName"`
+	PostgresqlImgTag string `json:"postgresqlImgTag"`
+	PostgresqlMaxConnections string `json:"postgresqlMaxConnections"` 
+	PostgresqlMemLimit string `json:"postgresqlMemLimit"` 
+	PostgresqlMemReq string `json:"postgresqlMemReq"`
+	PostgresqlSharedBuffers string `json:"postgresqlSharedBuffers"`
+
+	EncryptionKey string `json:"encryptionKey"`
+
+
+
+}
+
+// ManageiqStatus defines the observed state of Manageiq
+// +k8s:openapi-gen=true
+type ManageiqStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	HttpdISSetup bool `json:"httpdIssetup"`
+	MemcachedISSetup bool `json:"memcachedIssetup"`
+	PostgresqlISSetup bool`json:"postgresqlIssetup"`
+	OrchestratorISSetup bool `json:"orchestratorIsrunning"`
```

Comment:
Why `running` here and `setup` everwhere else?
Also the capitalization here is in consistent.

I would think this would be the format:
```go
OrchestratorIsSetup bool `json:"orchestratorIsSetup"`
```